tests/Behat/FeatureContext.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_editorial\Behat;
+
+use Drupal\DrupalExtension\Context\RawDrupalContext;
+use Behat\Mink\Exception\ExpectationException;
+use Behat\Mink\Element\NodeElement;
+use Behat\Gherkin\Node\TableNode;
+use PHPUnit\Framework\Assert;
+
+/**
+ * Defines step definitions that are generally useful in this project.
+ */
+class FeatureContext extends RawDrupalContext {
+
+  /**
+   * Checks that the given select field has the options listed in the table.
+   *
+   * @Then I should have the following options for :select select:
```

Comment:
for *the* :select select :)

Also can we maybe put a little doc on how this step would be used? Kinda like some of the methods inside `Drupal\DrupalExtension\Context\DrupalContext`?